bitnami/thanos/templates/querier/psp.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.querier.enabled .Values.querier.psp.create -}}
```

Comment:
The conditions to enable this for other charts is usually shown like this:

```
{{- if and .Values.rbac.create .Values.rbac.pspEnabled }}
```

Is there any reason for following a different format (via .Values.[component].psp.create)? For instance, have a look at [Prometheus](https://github.com/bitnami/charts/tree/master/bitnami/kube-prometheus/templates/prometheus).

Reply:
@marcosbc the reason being in prometheus is just one component, here we have multiple components `querier`, `ruler` etc. Followed the same approach being done for PDB

https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/querier/pdb.yaml#L1

Reply:
I've changed the condition for psp to be `pspEnabled` to make it consistent.

Reply:
Thanks!